installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+[id="preparing-to-install-with-agent-based-installer"]
+= Preparing to install with the Agent-based installer
+include::_attributes/common-attributes.adoc[]
+:context: preparing-to-install-with-agent-based-installer
+
+toc::[]
+
+== About the Agent-based installer
+Agent-based installer is a combination of the Assisted Installer (AI) technology and the installer-provisioned infrastructure(IPI). It is a subcommand that resides in the {product-title} installer and also utilizes the {product-title} installer for the input asset management.
+It generates a bootable image containing all of the information required to deploy an {product-title} cluster.
+
+This subcommand generates a live {op-system-first} ISO. The live ISO runs all of required components that are running in containers pulled from the {product-title} release payload.
+The Agent-based installer also utilizes Zero Touch Provisioning (ZTP) custom resources. ZTP allows you to provision new edge sites with declarative configurations of bare-metal equipment at remote sites.
```

Comment:
Seems redundant to refer to both "edge sites" and "remote sites" in same sentence.